packages/rum-core/src/browser/scrollHeightObservable.ts

```diff
@@ -0,0 +1,15 @@
+import { Observable, setInterval, clearInterval } from '@datadog/browser-core'
+
+export const SCROLL_HEIGHT_OBSERVABLE_INTERVAL_MS = 500
+
+export function createScrollHeightObservable() {
+  const observable = new Observable<number>(() => {
+    const setIntervalId = setInterval(() => {
+      observable.notify(Math.round((document.scrollingElement || document.documentElement).scrollHeight))
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You could factorise `document.scrollingElement || document.documentElement).scrollHeight` with the one in `computeScrollValues()`